.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+### Description
+
+<!--- Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change. --->
+
+### Related Issue 
+- issue goes here
+
+Fixes # [ISSUE]
+<!-- Please Mention the issue number as ISSUE #(Issue Number)
+
+
+### Type of Change:
+
+<!--- **Delete irrelevant options.** --->
+
+- Code
+- User Interface
+- New Feature
+- Documentation
+- Testing
```

Comment:
How about checkboxes? Is it good to put here?

Reply:
Yeah we can do it ...I will  do all changes you asked for